jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/resources/org/jbpm/workbench/pr/client/i18n/Constants_es.properties

```diff
@@ -105,7 +105,8 @@ Loading=Cargando...
 VariableValueUpdated=Variable {0} actualizada
 Ok=Aceptar
 Close=Cerrar
-Correlation_Key=Llave de correlación
+Correlation_Key=Código de negocio
```

Comment:
Avoid adding language specific changes by directly editing these files. These are overridden by syncing with translations stored in Zanata.